ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/wan_route_servers.schema.yml

```diff
@@ -25,9 +25,9 @@ keys:
         hostname:
           type: str
           description: Route-Reflector hostname.
-        router_id:
+        vtep_ip:
           type: str
-          description: Route-Reflector router id.
+          description: Route-Reflector VTEP IP Address.
```

Comment:
should we indicate this is the one under DPS interface for clarity for users?

Reply:
```suggestion
          description: Route-Reflector VTEP IP Address. This would normally be the IP address under `interface Dps1`.
```

Reply:
@gmuloc is this clearer?